code/modules/spells/spell_types/lichdom.dm

```diff
@@ -53,6 +53,7 @@
 		marked_item.name = "ensouled [marked_item.name]"
 		marked_item.desc += "\nA terrible aura surrounds this item, its very existence is offensive to life itself..."
 		marked_item.add_atom_colour("#003300", ADMIN_COLOUR_PRIORITY)
+		marked_item.AddComponent(/datum/component/stationloving, FALSE, TRUE)
```

Comment:
does stationloving work if the object is giving the component while it's already off of the station (and the object isn't moved)?